Victoria & Albert Museum.js

```diff
@@ -0,0 +1,198 @@
+{
+	"translatorID": "56fc13bf-411f-4409-9a61-3d334d76763b",
+	"label": "Victoria & Albert Museum",
+	"creator": "V&A Digital Media & Publishing",
+	"target": "^https?://collections\\.vam\\.ac\\.uk",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-02-12 14:45:24"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 V&A Museum
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes("/item") != -1) {
+		return "artwork";
+	}
+	else if (url.includes("/search") != -1) {
+		return "multiple";
+	}
+	return false;
+}
+
+function scrape(doc, url) {
+	var id = url.match(/item\/(O[0-9]+)/i)[1];
+
+	var apiUrl = "https://api.vam.ac.uk/v2/museumobject/" + id;
+	Zotero.Utilities.doGet(apiUrl, function (text) {
+		var museumobject = JSON.parse(text);
+		var data = museumobject.record;
+		var item = new Zotero.Item("artwork");
+		if (data.titles.length > 0) {
+			item.title = data.titles[0].title;
+			item.shortTitle = data.objectType;
+		}
+		else if (data.objectType !== "") {
+			item.title = data.objectType;
+			item.shortTitle = data.objectType;
+		}
+
+		item.abstractNote = data.briefDescription;
+		item.medium = data.materialsAndTechniques;
+		item.artworkSize = data.dimensionsNote;
+		if (data.productionDates.length > 1) {
+			item.date = data.productionDates[0].date.text + " and further dates";
+		}
+		else if (data.productionDates.length > 0) {
+			item.date = data.productionDates[0].date.text;
+		}
+
+		item.archive = data.collectionCode.text;
+		item.libraryCatalog = 'Victoria & Albert Museum';
+		item.callNumber = data.accessionNumber;
+		item.rights = data.creditLine;
+		item.url = url;
+
+
+		var artistMakerPerson = data.artistMakerPerson;
+		for (var i = 0; i < artistMakerPerson.length; i++) {
+			// This is not ideal and assume surname, firstnames convention is correct
+			var fullName = artistMakerPerson[i].name.text;
+			var lastComma = fullName.lastIndexOf(",");
```

Comment:
This logic can be replaced by `ZU.cleanAuthor(fullName, "author", fullName.includes(","))`.

Reply:
Don't quite follow how much this handles